### PR TITLE
adding franka experiment hyperparameters

### PR DESCRIPTION
**Pull request by aravindr93**

### CODE REVIEW

**Comment by girifb** [comment]
I doubt the issues we are seeing have to do with hyperparameters. This is the config i have been using. As you can see `# RL parameters (all params related to PG, value function etc.)` are identical between yours and mine. Only difference i see (discounting hydra stuff), are:

```
save_freq         :   5
eval_rollouts     :   4
```

I am not sure these impact training perf that much right ?

Here is the full config:

```
default:
    - override hydra/launcher: local
    - override hydra/output: local

# general inputs
env               :   kitchen_close-v3     # placeholder name
algorithm         :   NPG
seed              :   123
sample_mode       :   trajectories
rl_num_traj       :   20
rl_num_samples    :   0                       # will be ignored when sample_mode=trajectories
num_cpu           :   4
rl_num_iter       :   200
save_freq         :   5
eval_rollouts     :   4
exp_notes         :  'mj_envs commit 750c0c41cd3ff5829bc09d734b635e9f6a029e34; mjrl v2'

# RL parameters (all params related to PG, value function etc.)
policy_size       :   (64, 64)
init_log_std      :   -0.1
min_log_std       :   -1.0
vf_hidden_size    :   (128, 128)
vf_batch_size     :   64
vf_epochs         :   2
vf_learn_rate     :   1e-3
rl_step_size      :   0.05
rl_gamma          :   0.995
rl_gae            :   0.97

# Algorithm hyperparameters : if alg requires additional params, can be specified here (or defaults will be used)

alg_hyper_params  :   {'device': 'cpu'}

# general outputs
job_name          :   'job'

hydra:
    job:
        name: train_${algorithm}_${env}
```

**Comment by aravindr93** [comment]
@girifb, can you check your `rl_num_traj = 20` parameter again? I see a major difference between yours and mine (20 vs 200). This can make a massive difference in RL experiments. Aside from that everything else looks reasonable.

File: mjrl_dev/hydra/config/franka_mjrl_config.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    - override hydra/output: local
+
+# general inputs
+env               :   kitchen-v3              # placeholder name
+algorithm         :   NPG
+seed              :   123
+sample_mode       :   trajectories
+rl_num_traj       :   200
+rl_num_samples    :   0                       # will be ignored when sample_mode=trajectories
+num_cpu           :   10
```
Comment (vikashplus) [inline mjrl_dev/hydra/config/franka_mjrl_config.yaml:12]: FAIR cluster nodes have many cores. we can definitely bump this up a notch

File: mjrl_dev/hydra/hydra_mjrl_launcher.py [hunk 2]
```diff
@@ -11,7 +11,7 @@
 # ===============================================================================
 # Process Inputs and configure job
 # ===============================================================================
-@hydra.main(config_name="hydra_npg_config", config_path="config")
+@hydra.main(config_name="franka_mjrl_config", config_path="config")
```
Comment (vikashplus) [inline mjrl_dev/hydra/hydra_mjrl_launcher.py:14]: I'd prefer if we keep the default to point to the tutorial `hydra_npg_config`.
Let's move this into a command line override.

File: mjrl_dev/hydra/train_kitchen_tasks_v3.sh [hunk 3]
```diff
@@ -0,0 +1 @@
+python hydra_mjrl_launcher.py --multirun hydra/launcher=slurm hydra/output=slurm env=kitchen_micro_open-v3,kitchen_rdoor_open-v3,kitchen_ldoor_open-v3,kitchen_sdoor_open-v3,kitchen_light_on-v3,kitchen_knob4_on-v3,kitchen_knob3_on-v3,kitchen_knob2_on-v3,kitchen_knob1_on-v3
```
Comment (vikashplus) [inline mjrl_dev/hydra/train_kitchen_tasks_v3.sh:1]: Add a command line override to point to `franka_mjrl_config`

**Review by vikashplus — COMMENTED** [review]
Left a few small comments, otherwise looks good.

**Comment by vikashplus** [comment]
@girifb, I agree with @aravindr93  here 
Let's definitely change to followings in addition to the parameters that aravind provided. These can have big differences in the training.
```
rl_num_traj       :   200
num_cpu           :   24
rl_num_iter       :   1000
```

**Comment by aravindr93** [comment]
@vikashplus  We define max cpu to be 20 in the hydra config. Scaling doesn't work linearly beyond a point, and starting new processes only to rollout 5-10 trajectories may not be worth it. For num_traj=200, I think num_cpu=10 is the sweet-spot which is I why stuck with it.

**Comment by vikashplus** [comment]
That seems sensible. Lets address the other comments I mentioned and merge this soon.

**Pull request merged**